index.html

```diff
@@ -1,82 +1,327 @@
+                            <div class="work-info">
+                                <ul>
+                                    <li class="intro">FACEBOOK</li>
+                                    <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p>
+                                Experimental content creation feature that allows users to add to an existing story over
+                                the course of a day without spamming their friends.
+                            </p>
+
+                            <div class="tags">
+                                <ul>
+                                    <li>html</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                </ul>
+                            </div>
+
+                            <a href="#" class="btn-one">See project</a>
+                        </li>
+                    </ul>
+                </div>
+            </li>
+            <li class="work">
+                <div class="work-container">
+                    <ul class="work-pro">
+                        <li>
+                            <img src="./image/portfolio3.png" alt="Several hands holding beer glasses">
+                        </li>
+                        <li class="work-description">
+                            <h1><a href="">Facebook 360</a></h1>
+                            <div class="work-info">
+                                <ul>
+                                    <li class="intro">FACEBOOK</li>
+                                    <li> <i class="fa fa-circle"></i> Full Stack Dev</li>
+                                    <li><i class="fa fa-circle"></i> 2015</li>
+                                </ul>
+                            </div>
+                            <p>
+                                Exploring the future of media in Facebook's first Virtual Reality app; a place to
+                                discover and enjoy 360 photos and videos on Gear VR.</p>
+
+                            <div class="tags">
+                                <ul>
+                                    <li>html</li>
+                                    <li>css</li>
+                                    <li>javascript</li>
+                                </ul>
+                            </div>
+
+                            <a href="#" class="btn-one">See project</a>
+                        </li>
+                    </ul>
+                </div>
+            </li>
 
+            <li class="work">
+                <div class="work-container">
+                    <ul class="work-pro">
+                        <li>
+                            <img src="./image/portfolio4.png" alt="Several hands holding beer glasses">
```

Comment:
- [ ] Your logo image is not displayed, because you're using the wrong path for your image, kindly fix this, by changing the path to ```"../Image/portfolio4.png" ```